Dockerfile

```
@@ -124,6 +124,7 @@ ARG USER_ID=0
 RUN if [ "$USER_ID" != 0 ]; then \
         groupadd --gid "$USER_ID" qdrant; \
         useradd --uid "$USER_ID" --gid "$USER_ID" -m qdrant; \
+        mkdir snapshots storage; \
```

We might want `-p` here to prevent errors if they somehow already exist.

```suggestion
        mkdir -p snapshots storage; \
```